lib/common/grpc-service.js

```diff
@@ -325,7 +325,8 @@ GrpcService.prototype.requestStream = function(protoOpts, reqOpts) {
 
     request: function() {
       return service[protoOpts.method](reqOpts, grpcOpts)
-        .on('status', function(status) {
+        .on('metadata', function() {
+          var status = this.received_status || { code: 0 };
```

Comment:
@murgatroid99 -- is this a reliable way to get the status? Maybe it should be emitted with the metadata event?